### ISSUE TITLE
pull access denied for dcape-compose | Error: No such object: dcape-compose

### ISSUE THREAD

**Issue by Alieramada** [dopos/dcape#41]
Добрый день, не получается устанавливать приложения адаптированные под dcape (dcape-app-nginx-sample, nextcloud). Везде одна и та же картина.

В варианте с деплоем через cicd после  этапа "Test Delivery" в Woodpecker на этапе деплоя возникает ошибка:
`Error response from daemon: pull access denied for dcape-compose, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`. В enfist пусто.

При попытке сделать всё "ручками", после клонирования репозитория и перехода в его дерикторию, когда вводишь make config возникает ошибка:
`Error: No such object: dcape-compose
Error: No such object: dcape-compose
Error: No such object: dcape-compose
/opt/dcape/Makefile.app:58: /Makefile.common: Нет такого файла или каталога
make: *** Нет правила для сборки цели «/Makefile.common».  Останов.
`
Makefile.common в /opt/dcape присутствует,  DCAPE_ROOT         ?= /opt/dcape в Makefile.app прописан.

**Comment by LeKovr** [comment]
Спасибо за информацию, похоже в документации есть недоработка. Пока мы ищем и исправляем, просто выполните в `/opt/dcape` команду `make build-compose`

**Comment by Alieramada** [comment]
Да, спасибо большое, заработало! Будем пробовать деплоить и адаптировать.

**Comment by LeKovr** [comment]
Цель `build-compose` добавлена в процесс install: cf850c27
Теперь эта проблема возникать не будет
